andela/wg-rogue-one
# #157731630 Creates a get exercises on hover over image body part feature

### What does this PR do?
This PR creates a feature that enables users to hover over a body part in the image
and display the exercises for that body part.

Add event handlers to `wger/exercises/templates/muscles/overview.html`
to get the required target on mouse hover over image, and display the relevant body exercises
Add flip-front and flip-back button to enable users flip the image to the back side,
in case you want to hover over the back muscles
Add default svg css to `wger/core/static/css/workout-manager.css` to hide the overlapping svg
images with the targeted ids and position the image over the background image.
Add a new file muscles.js with the muscles svg image variables to `wger/exercises/static/js/muscles.js`

#### Description of Task to be completed?
The user should be able to move the mouse over a body part on the image and get exercises for that body part.

#### How should this be manually tested?
1: Clone the application and install the requirements in your virtual environment.
2: Update the database settings in the `settings.py` file with your postgres user, password and database name.
3: Run the application with `python manage.py runserver`.
4: Register and login. 
5: Hover over `Exercises` in the navbar menu and select `Muscle overview`.
6: Hover over the body parts in the image. Sample image shown below: 
<img width="1186" alt="screen shot 2018-06-06 at 14 51 14" src="https://user-images.githubusercontent.com/25340897/41036575-2c7d565e-6999-11e8-9e51-5a48338939a2.png">

7: Expected response: The application should display the exercises related to the muscle you hovered over.
8: The application should also auto highlight the muscle name related to the muscle you hovered over. Sample response shown below.

<img width="1152" alt="screen shot 2018-06-06 at 14 47 17" src="https://user-images.githubusercontent.com/25340897/41036393-91f254f4-6998-11e8-9f1a-b09df8b4d7dc.png">
The exercises are displayed in the far right section.

9: Click the `front` or `back` buttons to flip the image.

#### Any background context you want to provide?
This feature was not available.

#### What are the relevant pivotal tracker stories?
#157731630
